.github/workflows/owasp-dependency-check-daily.yml

```diff
@@ -21,6 +23,7 @@ jobs:
 
       - name: Set up gradle
         uses: gradle/actions/setup-gradle@v4
+
```

Comment:
On line 30 there's an "Upload report" step that may need write permissions.

Reply:
it's only uploading as an attachment to the action run (I double checked this is still working in instrumentation repo)